src/client.js

```diff
@@ -133,52 +133,48 @@ assign(Client.prototype, {
     })
   },
 
-  stream: function(connection, obj, stream, options) {
+  stream(connection, obj, stream, options) {
     if (typeof obj === 'string') obj = {sql: obj}
     this.emit('query', assign({__knexUid: connection.__knexUid}, obj))
     debugQuery(obj.sql)
     return this._stream.call(this, connection, obj, stream, options)
   },
 
-  prepBindings: function(bindings) {
+  prepBindings(bindings) {
     return bindings;
   },
 
-  wrapIdentifier: function(value) {
-    return (value !== '*' ? '"' + value.replace(/"/g, '""') + '"' : '*')
+  wrapIdentifier(value) {
+    return (value !== '*' ? `"${value.replace(/"/g, '""')}"` : '*')
   },
 
-  initializeDriver: function() {
+  initializeDriver() {
     try {
       this.driver = this._driver()
     } catch (e) {
-      helpers.exit('Knex: run\n$ npm install ' + this.driverName + ' --save' + '\n' + e.stack)
+      helpers.exit(`Knex: run\n$ npm install ${this.driverName} --save\n${e.stack}`)
     }
   },
 
   Pool: Pool2,
 
-  initializePool: function(config) {
+  initializePool(config) {
     if (this.pool) this.destroy()
     this.pool = new this.Pool(assign(this.poolDefaults(config.pool || {}), config.pool))
     this.pool.on('error', function(err) {
-      helpers.error('Pool2 - ' + err)
+      helpers.error(`Pool2 - ${err}`)
     })
     this.pool.on('warn', function(msg) {
-      helpers.warn('Pool2 - ' + msg)
+      helpers.warn(`Pool2 - ${msg}`)
     })
   },
 
-  poolDefaults: function(poolConfig) {
-    var dispose, client = this
-    if (poolConfig.destroy) {
-      helpers.deprecate('config.pool.destroy', 'config.pool.dispose')
-      dispose = poolConfig.destroy
-    }
+  poolDefaults(poolConfig) {
+    const client = this
```

Comment:
@jurko-gospodnetic Yeah, so I discovered this because ESLint was complaining about `dispose` never being used. I traced the history of this feature back and it seems very old (the history was disrupted by the introduction of Babel, I think over a year ago). I think it's safe to remove.

Who knows what `dispose` was meant to do. Whatever it was it wasn't doing it.